.gitignore

```diff
@@ -7,3 +7,4 @@ venv/
 .venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
There is an extra space before `db.sqlite3`. This might prevent Git from correctly ignoring the file. Please remove the space to ensure it is properly ignored.